### DISCUSSION TITLE
How to run cronjobs?

### DISCUSSION THREAD

**Discussion by Necriso** [php/frankenphp#152] [Q&A] [answered]
Hello

I got Symfony to work with frankenphp with Docker and a MySQL Database
Now I think about to start commands via CronJob, but have no idea how to realize that.
Anyone has an idea how to do that?

**Comment by withinboredom** [comment, marked as answer]
You'd do a cronjob just like anything else:

```sh
php vendor/bin/console my:command
```

If you're using Docker, you can do the same from the host:

```sh
docker run --rm --name cron-job --network my-network my-image php vendor/bin/console my:command
```

`--rm` and `--name` would delete your logs, but ensures that exactly one is running at any given time. You can still get the logs depending on how you have logging set up on the host.
